javaScript-week2/Event-application.js

```diff
@@ -0,0 +1,21 @@
+function getEventWeekday(numOfDaysToBeAdded) {
+    const date = new Date(); //gives present date
+    const dayNum = date.getDay(); //is zero based, returns a number from 0-6 and weekday No. in the date.(starts from sunday)
+    const namesOfDays = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+    ];
+    const i = numOfDaysToBeAdded + dayNum; //retuns this if the sum is less than 7
+    if (i >= 7) {
+      let r = i % 7; //retuns this if the sum is more or equal to 7 (dividing by 7 an using the remainder)
+      return namesOfDays[r];
+    } else {
+      return namesOfDays[i];
+    }
```

Comment:
You've overcomplicated it a bit here. What would happen if `i = 0`, or `i = 1`? Then you get the wrong days.
Skipping the conditional logic and directly writing it like this will work correctly:
```
return namesOfDays[i % 7]
```

Reply:
ok, changing it